File: OrderNet-Server/src/main/java/bitc/fullstack503/ordernetserver/dto/app/OrderRequestDTO.java
package bitc.fullstack503.ordernetserver.dto.app;

import lombok.Data;

import java.util.List;

@Data
public class OrderRequestDTO {
  // 생성한 주문번호
  private String orderId;

  // 주문 일자
  private String orderDate;

  // 주문 납품 일자
  private String orderDueDate;

  // 주문 총 금액
  private int orderPrice;

  // 대리점 ID
  private String branchId;

  private List<OrderItemDTO> items;
}

File: OrderNet-APP/app/src/main/java/bitc/fullstack/app/dto/OrderRequestDTO.kt
package bitc.fullstack.app.dto

import java.math.BigDecimal


data class OrderRequestDTO(
    // 주문일자
    val orderDate: String,

    // 납품일자
    val orderDueDate: String,

    // 총 금액
    val orderPrice: Int,

    // 대리점 ID
    val branchId: String,

    // 주문 항목 (OrderItem 리스트)
    val items: List<OrderItemDTO>
)
